simuq/aais/rydberg2d.py

```diff
@@ -15,61 +16,45 @@ def ham_sum(hlist) :
     for i in range(n) :
         ham += hlist[i].ham
 
-    from simuq.hamiltonian import TIHamiltonian
     return TIHamiltonian(sites_type, ham)
 
-def GenMach(n = 3, inits = None) :
-    Rydberg = QMachine()
+class Rydberg2DQMachineFactory(QMachineFactory):
+    @staticmethod
+    def generate_qmachine(n=3, inits=None, *args, **kwargs):
+        rydberg = QMachine()
 
-    C6 = 862690 * 2. * np.pi
+        C6 = 862690 * 2. * np.pi
```

Comment:
This should be a constant outside of the method, and Python convention says it should be `C_6`